hir-analysis/src/liveness.rs

```diff
@@ -13,6 +13,12 @@ use rustc_hash::FxHashMap;
 
 use super::{ControlFlowGraph, DominatorTree, LoopAnalysis};
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+struct BlockInfo {
+    max_register_pressure: u16,
```

Comment:
Yeah, I'm not sure if it will ever come up in practice that we need it (rather than `max_operand_stack_pressure`), but it is essentially free to compute it, so 🤷‍♂️